target/linux/mediatek/filogic/base-files/etc/board.d/01_leds

```diff
@@ -54,6 +54,12 @@ netgear,wax220)
 	ucidef_set_led_netdev "wlan2g" "WLAN2G" "blue:wlan2g" "phy0-ap0"
 	ucidef_set_led_netdev "wlan5g" "WLAN5G" "blue:wlan5g" "phy1-ap0"
 	;;
+openembed,som7981)
+	ucidef_set_led_netdev "lanact" "LANACT" "green:lan" "eth1" "rx tx"
+	ucidef_set_led_netdev "lanlink" "LANLINK" "amber:lan" "eth1" "link"
+	ucidef_set_led_netdev "wlan2g" "WLAN2G" "red:wlan-1" "phy0-ap0"
```

Comment:
Does this also work when you define `linux,default-trigger = "phy0tpt";` in the led definition in the device tree instead?

Reply:
Thank you for the suggestions, it works fine!